### PR TITLE
Fixed textarea disabled and added passing test

### PR DESCRIPTION
**Pull request by joshpfosi**
Modeling the `TextArea` component off of the function `TextInput` component solves this problem.

### CODE REVIEW

File: addon/components/em-text.js [hunk 1]
```diff
@@ -10,12 +10,13 @@ Syntax:
  */
 export default FormGroupComponent.extend({
   controlView: Ember.TextArea.extend(ControlMixin, {
+    attributeBindings: ['placeholder', 'required', 'autofocus', 'disabled'],
```
Comment (pedrokiefer) [inline addon/components/em-text.js]: Does the test pass without this line? TextArea already defines this bindings, so this should not be needed.

**Comment by pedrokiefer** [comment]
Just tested here, it passes, can you update de PR removing that line?

**Comment by joshpfosi** [comment]
Done!

**Comment by pedrokiefer** [comment]
Thanks! :+1:

**Pull request merged**